README.md

```diff
@@ -0,0 +1,30 @@
+# LaTeX Project-Manager
+*A project-manager tool to speed up creation process for note-taking*
+---
+## **Description**
+LPM (LaTeX Project Manager) is a simple project manager to generate a new LaTeX project from a template.
+
+## Requirements
+- Python 3.9 at least
+- `art` library (`python3 -m pip install art`)
+
+Tested on *Ubuntu 22.04 LTS*. 
+  
+## Installation
+```
+git clone https://github.com/iScsc/LaTeX-Project-Manager.git
+cd ./LaTeX-Project-Manager/settings
+```
+- Change the "PathToSource" with the *absolute* path to the `source_TeX` folder in the `/settings/config.json` file.
+```
+cd ../src
+```
+- Change the path with the *absolute* path to the `config.json` file. 
+```python
+SettingsLaunch = Settings(r"/home/nlesquoy/ghq/LaTeX-Project-Manager/settings/config.json")
```

Comment:
this is a remark for the whole code base: `python` has a guideline called `PEP8`, with both code and style rules.
these rules does not HAVE to be followed, but the thing that everyone, me included, use them and a code which violates these rules is super hard to read and appreciate, even if great...

one of these rules is to prefer _snake\_case_ over _CamlCase_ for everything, except class names, including variables, functions and files :+1: 

i recommend updating the code base in that direction

and as a general rule of thumb, you can use `flake8` and `black` to lint and format your `python` source files respectively :+1:

Reply:
Indeed I will use `flake8` to clean the code. I will also rewrite the docstring for another documentation tool (the current tool used here is `sphinx`).

Reply:
`sphinx` is fine, but change the framework if you fell more comfortable with another one :ok_hand:

Reply:
i also quite like the [Google Style Python Docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) if you do not already know it :wink: